app/models/agents/webhook_agent.rb

```diff
@@ -100,8 +100,14 @@ def receive_web_request(request)
           return ["Not Authorized", 401]
         end
 
-        JSON.parse(response.body)['success'] or
-          return ["Not Authorized", 401]
+        body = JSON.parse(response.body)
+        if body['score'].presence
+          body['score'] > 0.5 or
```

Comment:
Like you said it would be nice to make the score configurable, `interpolated['score_threshold'].to_f` would return the Agent configuration value of `score_threshold`.

Reply:
> Like you said it would be nice to make the score configurable, `interpolated['score_threshold'].to_f` would return the Agent configuration value of `score_threshold`.